opentelemetry-sdk/Cargo.toml

```diff
@@ -28,6 +28,7 @@ tokio = { workspace = true, features = ["rt", "time"], optional = true }
 tokio-stream = { workspace = true, optional = true }
 http = { workspace = true, optional = true }
 tracing = {workspace = true, optional = true}
+async-compat = { version = "0.2"}
```

Comment:
This package is for marshalling async events from futures runtime to tokio /async-std. Its dependency can be eliminated by integrating the necessary code from this package, as it appears to be relatively small.